mkt/extensions/validation.py

```diff
@@ -19,6 +19,8 @@ class ExtensionValidator(object):
     error.
     """
     errors = {
+        'AUTHOR_TOO_LONG': _(
+            u'The `author` property cannot be longer than 255 characters.'),
```

Comment:
Sure. Do you have a link for the decision ? I was looking for one and couldn't find any.